bundles/org.eclipse.text.quicksearch/src/org/eclipse/text/quicksearch/internal/ui/QuickSearchDialog.java

```diff
@@ -1030,47 +1168,6 @@ private StyledString highlightMatches(String visibleText) {
 		return styledText;
 	}
 
-// Version using sourceviewer
```

Comment:
do we know why this was discarded in the first place?

Reply:
It was already there (see [1st commit](https://github.com/eclipse-platform/eclipse.platform.ui/commit/e8b733007b85e76ddab4836dad7175c5012cc000#diff-4a0fd7cc5b0c2be5fdf99d5e66762c8138259bb36b340c76e7ce14067f161c7f)) when QuickSearchDialog code was contributed from [Spring Tools suite](https://github.com/spring-attic/eclipse-integration-commons/blame/main/org.springsource.ide.eclipse.commons.quicksearch/src/org/springsource/ide/eclipse/commons/quicksearch/ui/QuickSearchDialog.java#L1144)
Form what I can tell it was always just a protoype code - when I tried it uncommented it looked just like that.